README.md

```diff
@@ -2,7 +2,7 @@
 
 <!-- MDOC !-->
 
-An HTTP client with a focus on performance.
+An HTTP client with a focus on performance, built on top of `Mint` & `NimblePool`.
```

Comment:
@wojtekmach when I generate the docs locally, NimblePool links correctly, but Mint does not. Any idea why?

Reply:
oh, that caught me by surprise too. The reason is there's no module `Mint`. (there's `Mint.HTTP`, etc)

Reply:
oh of course, that does make sense. Hm I wonder if I should link to `Mint.HTTP` or use a normal markdown link to `https://hexdocs.pm/mint` instead 🤔

Reply:
yeah, both are fine, don't know which is more appropriate!